classes/Thankable/Thankable.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Claromentis\ThankYou\Thankable;
+
+class Thankable
+{
+	/**
+	 * @var string $name
```

Comment:
Same as `Tag` and `ThankYou`, variable names aren't needed in the docblocks for class properties.

Reply:
Fixed in 396cdf3be76461324e5cbc0f6f812b0af263f5be